KsApi/models/graphql/adapters/CreatePaymentSourceEnvelope+CreatePaymentSourceMutation.Data.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+import ReactiveSwift
+
+extension CreatePaymentSourceEnvelope {
+  static func from(_ data: GraphAPI.CreatePaymentSourceMutation.Data) -> CreatePaymentSourceEnvelope? {
+    guard let createdPaymentSource = data.createPaymentSource,
+      createdPaymentSource.isSuccessful,
```

Comment:
Curious if we need to perform this check here?

Reply:
yea, there's no description of this flag on GraphiQL, I just assumed it meant the mutation was successful. Seemed to be returning `true` for all mutations I tried. Presumably if a mutation fails on backend, this flag is `true`. It was being parsed in the GQL schema, so I thought might as well use it here.

Reply:
I can remove it though.

Reply:
I can remove it, mostly just used it for testing.

Reply:
Oops, I meant to say it's false if the query fails, I remembered trying invalid data on a mutation here (ie. tokenId was empty string) returned this flag as `false`. So it is useful for any reason the backend didn't make the requested change. Can be like a early exit before trying to parse all of the data.